src/legacy/core_plugins/expressions/public/np_ready/public/loader.ts

```diff
@@ -135,6 +135,9 @@ export class ExpressionLoader {
     this.setParams(params);
     this.dataHandler = new ExpressionDataHandler(expression, params);
     const data = await this.dataHandler.getData();
+    if (data.type === 'error' && data.error.name === 'AbortError') {
+      return;
```

Comment:
Someone more familiar with expressions should chime in here. I'm wondering what we should do in the case of an abort? Should we go back to the previous state? Not sure whether a plain return is the right thing here.

Reply:
i don't think we should do this and should not make abort error a special case in how the expression service behaves.

Reply:
also i think there are two scenarious when AbortError can happen: (or maybe even more)
- abort gets called from the outside (dashboard or something). in this case we should show the error, as its popssible visualization didn't even finish its first render, so there might be nothing else to show. Also we don't know if (and when) we'll get a chance to do the request again.
- we have request in progress but then we get another update call before the first request completed. we (internally) abort the first one and start a new one. This is where we don't want to show the error.